packages/fast-components-foundation-react/src/foundation/foundation.spec.tsx

```diff
@@ -233,6 +244,9 @@ describe("unhandledProps", () => {
         unhandledPropsTestComponentDirty = new ReactTestUtils.renderIntoDocument(
             <UnhandledPropsTestComponent {...merge(cleanProps, dirtyProps)} />
         );
+        nullHandledpropsTestComponent = new ReactTestUtils.renderIntoDocument(
+            <UnhandledPropsOnlyTestComponent {...dirtyProps} />
+        );
```

Comment:
I think this can just be defined in your test. We don't need this setup before each test.

Reply:
That makes sense... push should be in